src/main.zig

```diff
@@ -34,7 +34,7 @@ const key_ctrl_w = 23;
 const key_esc = 27;
 const key_backspace = 127;
 
-fn linenoiseEdit(ln: *Linenoise, in: File, out: File, prompt: []const u8) !?[]const u8 {
+pub fn linenoiseEdit(ln: *Linenoise, in: File, out: File, prompt: []const u8) !?[]const u8 {
```

Comment:
These functions are intentionally not public as the API surface is intended to have only one entry point (`Linenoise.linenoise`).